forum/qa-plugin/user-activity-log/user-activity-log-search.php

```diff
@@ -0,0 +1,191 @@
+<?php
+
+class user_activity_search
+{
+    
+    private $searchArr;
+    private $dbResult;
```

Comment:
brakuje typowania

Reply:
Z tego co przeczytałem w dokumentacji i sprawdziłem u siebie typowanie dla atrybutów klas jest wprowadzone od wersji 7.4.0

>Type declarations can be added to function arguments, return values, and, as of PHP 7.4.0, class properties. They ensure that the value is of the specified type at call time, otherwise a [TypeError](https://www.php.net/manual/en/class.typeerror.php) is thrown.

[link do docsów](https://www.php.net/manual/en/language.types.declarations.php)
U nas w CONTRIBUTING.md jest napisane że korzystamy z wersji 7.1 co faktycznie jest prawdą( po wpisaniu `php -v` jako wersję php w dockerowym bashu)

Reply:
Racja, nie sprawdziłem.